features/gift_exchanges/challenge_yuletide.feature

```diff
@@ -448,7 +459,8 @@ Feature: Collection
   When I follow "Gifts"
   Then I should not see "Stargate Atlantis"
     And I should not see "myname" within "ul.gift"
-  When I go to the collections page
+  When all indexing jobs have been run 
```

Comment:
This can be removed now that `I go to the collections page` runs the indexing jobs.